File: TagMemo-java/app/src/main/java/com/timejh/tagmemo_java/model/Group.java
package com.timejh.tagmemo_java.model;

import io.realm.RealmList;
import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by tokijh on 2017. 4. 27..
 */

public class Group extends RealmObject {

    public String id;

    public String parentGroupId;
    public String title;
    public RealmList<HashTag> tags;
    public RealmList<GroupMemo> groupMemos;

    public String last_date;
    public boolean isValidated;
}

File: TabMemo-Swift/TabMemo-Swift/Group.swift
//
//  Group.swift
//  TabMemo-Swift
//
//  Created by 윤중현 on 2017. 5. 28..
//  Copyright © 2017년 윤중현. All rights reserved.
//

import RealmSwift

class Group: Object {
    dynamic var id: String = ""
    
    dynamic var parentGroupId: String = ""
    dynamic var title: String = ""
    let tags = List<HashTag>()
    let groupMemos = List<GroupMemo>()
    
    dynamic var last_date: String = ""
    dynamic var isValidated: Bool = false
}
